library_management/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from users.models import User
+
+
+class Book(models.Model):
+    name = models.CharField(max_length=100)
+    author = models.CharField(max_length=100)
+    publisher = models.CharField(max_length=100)
+    image = models.ImageField(upload_to='books/', null=True, blank=True)
+    quantity = models.PositiveIntegerField(default=0)
+
+
+class BookIssue(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    issued_date = models.DateField()
+    return_date = models.DateField()
+    returned = models.BooleanField(default=False)
+    reminder_sent = models.BooleanField(default=False)
+
+
+class BookRequest(models.Model):
+    REQUEST_STATUS_CHOICES = (
+        ('Pending', 'Pending'),
+        ('Approved', 'Approved'),
+        ('Rejected', 'Rejected'),
+    )
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    request_date = models.DateTimeField(auto_now_add=True)
+    status = models.CharField(
+        max_length=20, choices=REQUEST_STATUS_CHOICES, default='Pending')
+    rejection_reason = models.TextField(blank=True, null=True)
+
+    def __str__(self):
+        return f'{self.user.username} requests {self.book.name}'
```

Comment:
We can modify this table to store information about issued and returned books as well. This way we can get rid of `BookReturn` and `BookIssue`.

Reply:
Thanks for the feedback. I have updated the code.